Client/src/Pages/Settings/index.jsx

```diff
@@ -38,6 +39,8 @@ const Settings = ({ isAdmin }) => {
     ttl: checkTTL ? (checkTTL / SECONDS_PER_DAY).toString() : 0,
   });
   const [errors, setErrors] = useState({});
+  const delStatMntrInitState = { delMntrs: false, delSts: false };
```

Comment:
I think my previous comments disappeared =/
I believe keeping one state as an object could be good in this case for performance reasons. We are always having only one dialog open. So whenever we open one, we should close the other. If that is the case, instead of having to do: setState1(false), setState2(true), which would mean two rerenders and possible performance issues, we could keep them in only one state, do you agree?

Reply:
Also, I agree on the more descriptive names

Reply:
> I think my previous comments disappeared =/ I believe keeping one state as an object could be good in this case for performance reasons. We are always having only one dialog open. So whenever we open one, we should close the other. If that is the case, instead of having to do: setState1(false), setState2(true), which would mean two rerenders and possible performance issues, we could keep them in only one state, do you agree?

This is a good point, thanks for bringing that up :+1: Agreed, let's stick with the more complex state to avoid needless double render.  It may come in handy if we have more dialogs to open here in the future as well.

@jennifer-gan if you can just clean up the names then we're good to go on this one I think!